catalog/spatial/geocoding/spatial-geocoding-geocoder/src/main/java/org/codice/ddf/spatial/geocoder/GeoCoder.java

```diff
@@ -32,4 +34,14 @@
      * @throws GeoEntryQueryException if an exception occurs while querying the GeoNames resource
      */
     NearbyLocation getNearbyCity(String locationWKT) throws GeoEntryQueryException;
+
+    /**
+     * Retrieves the country code for a given location. The center point is used if {@code locationWKT}
+     * is a polygon.
+     *
+     * @param locationWKT WKT location for which to get the country code of
+     * @param radius      Radius in kilometers to search from the center of {@code locationWKT}
+     * @return a country code in ISO 3166-1 alpha-3 format or null if not found (for example, a location in the ocean)
+     */
+    Optional<String> getCountryCode(String locationWKT, int radius);
```

Comment:
Just an idea, but since we want to be strict about always returning a 3-letter country code, should be have a POJO that holds the code and enforces the format? That would better communicate the contract.